crates/project/src/lsp_store.rs

```diff
@@ -7376,6 +7385,16 @@ impl LspStore {
 
             let toolchain_store = this.update(cx, |this, cx| this.toolchain_store(cx)).ok()?;
             for (adapter, server, delegate) in servers {
+                // TODO
+                // For some reason, the Deno LSP cannot respond properly to textDocument/definition requests once it has received a workspace/didChangeConfiguration notification.
+                // This issue does not occur with external packages (e.g. deno:/https/jsr.io/%40fresh/core/2.0.0-alpha.37/src/plugins/fs_routes/mod.ts), but it does happen with built-in packages (e.g. deno:/asset/lib.deno.console.d.ts).
+                // As a result, the “Go to Definition” feature fails for something like console.log after sending workspace/didChangeConfiguration.
+                // So far, we haven’t been able to create a minimal, standalone reproducible example that clearly demonstrates this is a bug in the Deno LSP.
+                // For now, we’re simply skipping workspace/didChangeConfiguration entirely, though this is probably not the best approach.
+                if adapter.name().0.contains("deno") {
```

Comment:
This is an invasive change that also skips valid use cases, so I’ll let the team decide whether to proceed with it.